Source/msg.cpp

```diff
@@ -89,6 +89,10 @@ void PrePacket()
 				data += sizeof(*cmd);
 				spaceLeft -= sizeof(*cmd);
 				playerId = cmd->bPlr;
+				if (playerId > MAX_PLRS) {

```

Comment:
```suggestion
				if (playerId >= MAX_PLRS) {
```

ParseCmd doesn't validate playerId before trying to use it as an index into Players